crates/api_models/src/enums.rs

```diff
@@ -43,6 +43,7 @@ pub enum RoutingAlgorithm {
 #[serde(rename_all = "snake_case")]
 #[strum(serialize_all = "snake_case")]
 pub enum Connector {
+    Square,
```

Comment:
please maintain the alphabetical order wherever the enum is being used

Reply:
Also Comment this enum variant, as it will throw errors once merged in main

Reply:
Done.